appveyor.yml

```diff
@@ -33,7 +33,7 @@ test_script:
   # Change to a non-source folder to make sure we run the tests on the
   # installed library.
   - "cd C:\\"
-  - "python -c \"import nose; nose.main()\" -v -s joblib"
+  - "py.test --disable-pytest-warnings --pyargs joblib"
```

Comment:
`pytest > 3.0.0` raises more than 300 pytest warnings as `yield` based tests have been officially deprecated. Since they're expected to arise, and a long list messes up the CI log, I have disable them for a while, now from both Windows and Linux. I'll remove this once I parametrize yield based tests in subsequent PRs.